sdk/python/lib/pulumi/resource.py

```diff
@@ -261,6 +267,7 @@ def __init__(self,
                resource.
         """
         Resource.__init__(self, t, name, True, props, opts)
+        self.__pulumiType = t
```

Comment:
Is this change related to the work for this PR?

Reply:
I think you could argue no. We can pull it into a separate PR.